src/js/apis/admin/1.0.0/controller/controller.js

```diff
@@ -110,6 +111,69 @@ module.exports = {
 
     return request;
 
+  },
+
+
+  /**
+  *
+  * @function
+  * @name checkProjectionParameters
+  * @description Vérification des paramètres d'une requête sur /services/{service}/projections/{projection}
+  * @param {object} parameters - ensemble des paramètres de la requête ExpressJS
+  * @return {ProjectionRequest} request - Instance de la classe ProjectionRequest
+  *
+  */
+
+  checkProjectionParameters: function(parameters) {
+
+    LOGGER.debug("checkProjectionParameters()");
+
+    // Service
+    if (!parameters.service) {
+      throw errorManager.createError(" Parameter 'service' is invalid: there is no value", 400);
+    } 
+    
+    if (parameters.service === "") {
```

Comment:
Can this happen ? If yes, so we could let the 400.

Reply:
I added a trim to the service parameter so we can check for empty value. 
I kept the 400.